justfile

```diff
@@ -20,4 +20,4 @@ clippy:
     cargo clippy --all-features --tests -- -D warnings
 
 reset-db:
-    diesel migration revert --all --database-url=harbor.sqlite && diesel migration run --database-url=harbor.sqlite
+    cd harbor-client && diesel migration revert --all --database-url=harbor.sqlite && diesel migration run --database-url=harbor.sqlite
```

Comment:
@benthecarman I've been wondering about this command, is it right as the default used is `~/.harbor` ?

Reply:
Yeah but looks like this was incorrect even before this pr. Will make an issue, good catch.